content/docs/sidebar.json

```
@@ -520,6 +520,10 @@
     "label": "Python API Reference",
     "source": "api-reference/index.md",
     "children": [
+      {
+        "slug": "dvcfs",
+        "label": "DvcFileSystem"
```

Not seeing `dvcfs` as an actual identifier in docs or code. Maybe
```suggestion
        "slug": "dvcfilesystem",
        "label": "DvcFileSystem"
```
Or
```suggestion
        "slug": "dvc.fs",
        "label": "DvcFileSystem"
```